api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceService.java

```diff
@@ -78,4 +80,8 @@ UserRecentWorkspace updateRecentWorkspaces(
   UserRecentWorkspace updateRecentWorkspaces(Workspace workspace);
 
   boolean maybeDeleteRecentWorkspace(long workspaceId);
+
+  String getRegisteredUserDomainEmail();
```

Comment:
does this have to be public? It looks like all of the usages can be narrowed down to `WorkspaceServiceImpl`.

It is called from WorkspaceController but all 4 call sites pass in the same value so we can refactor the arguments to remove the registered users group and fetch it directly from `WorkspaceServiceImpl`.

Reduces # of arguments and reduces interface scope so I think it's a win-win

Reply:
Oh, yeah, good point.